src/Consoles/MainConsole.cs

```diff
@@ -19,11 +20,22 @@ public class MainConsole : Console
 
         private Point Middle { get; }
 
-        private Room testRoom = TestRooms.Room1;
+        public Room CurrentRoom { get; set; } = TestRooms.Room1;
 
         public MainConsole(int width, int height) : base(width, height)
         {
             Player = new Player("Player", Fraction.Warrior, 10);
+            foreach(var w in CurrentRoom.Entities)
+            {
+                w.Transform.Collision = new CollisionBox(
+                new Polygon(new List<Rectangle> { new Rectangle(0, 0, 1, 1) })
+                );
+            }
```

Comment:
Mój błąd nie jest to klasa anonimowa, chodziło mi o tę inicjalizację która nie wiedzieć dlaczego nie pozostawiała CollisionBoxa tak jak ustawił go konstruktor Entity, przeniosę konstruktor do tranforma